itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/AbstractNewQuarkusProjectTest.java

```diff
@@ -146,8 +154,35 @@ public static void checkProblemsView() {
 
 	@After
 	public void deleteProject() {
+		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
 		ProjectExplorer pe = new ProjectExplorer();
 		pe.open();
 		pe.deleteAllProjects(true);
+		new WaitWhile(new JobIsRunning(), TimePeriod.VERY_LONG);
+	}
+
+	public static void checkUrlContent() {
+		String urlContent = "";
+
+		try {
+			urlContent = getUrlContent("http://localhost:8080/hello");
+		} catch (Exception e) {
+			QuarkusCorePlugin.logException("Wrong URL!", e);
+		}
+
+		if (!(urlContent.equals("hello"))) {
+			throw new RedDeerException("Should be 'hello' !");
```

Comment:
instead of throwing RedDeerException, you can fail test instead: http://junit.sourceforge.net/javadoc/org/junit/Assert.html#fail(java.lang.String)